src/utils/config.ts

```diff
@@ -200,15 +200,14 @@ export async function validateSecureBootOptionAndWarn(
 		throw new ExpectedError(`Error: No device type provided`);
 	}
 	const sdk = getBalenaSdk();
-	const [osRelease] = await sdk.models.os.getAllOsVersions(slug, {
+	const [{ contract }] = await sdk.models.os.getAllOsVersions(slug, {
```

Comment:
This will explode w/ `trying to access property compose in undefined`.
I think it would be safer if we did v
```suggestion
	const [osRelease] = await sdk.models.os.getAllOsVersions(slug, {
```
and then the `if` will become `if (!osRelease?.contract) {`